DiscordBot.Modules/CommandModules/PollModule.cs

```diff
@@ -123,15 +123,48 @@ public async Task EndPollAsync(string idOrUrl, [Remainder] string message = "")
         private bool CheckPollMsg(IUserMessage msg)
         {
             //Console.WriteLine($"`{msg.Embeds.First().Fields.Count()}Desc");
+            if (msg.Content != "")
+            {
+                return false;
+            }
+
+            if (msg.Embeds.Count != 1)
+            {
+                return false;
+            }
+
             var emb = msg.Embeds.FirstOrDefault();
-            if ((msg.Content != "") ||
-                (msg.Embeds.Count != 1) ||
-                (emb.Title == "") ||
-                (emb.Footer.ToString() != $"{_prefix}poll") ||
-                (emb.Timestamp == null) ||
-                (emb.Description != null) ||
-                (emb.Fields.Count() != 1) ||
-                (emb.Fields.First().Value == "")) { return false; }
+
+            if (emb.Title == "")
+            {
+                return false;
+            }
+
+            if (emb.Footer.ToString() != $"{_prefix}poll")
+            {
+                return false;
+            }
+
+            if (emb.Timestamp == null)
+            {
+                return false;
+            }
+
+            if (emb.Description != null)
+            {
+                return false;
+            }
+
+            if (emb.Fields.Count() != 1)
```

Comment:
.Count() (die methode) nach Möglichkeit IMMER vermeiden, das heisst meistens nichts gutes. Wenn möglich .Count (Property), ansonsten .Any().
in dem Fall `!emb.Fields.Any()`

Reply:
Den und den nächsten check könnte man verbinden mit if(emb.Fields.Any(x => !string.IsNullOrEmpty(x.Value)))